packages/blockchain-wallet-v4-frontend/src/scenes/InterestHistory/DownloadTransactions/index.tsx

```diff
@@ -0,0 +1,176 @@
+import React from 'react'
+import { CSVLink } from 'react-csv'
+import { FormattedMessage } from 'react-intl'
+import { connect, ConnectedProps } from 'react-redux'
+import { flatten, map } from 'ramda'
+import { bindActionCreators, Dispatch } from 'redux'
+import styled, { css } from 'styled-components'
+
+import { HeartbeatLoader, IconButton } from 'blockchain-info-components'
+import { actions, selectors } from 'data'
+import { InterestHistoryCoinFormType } from 'data/components/interest/types'
+import { RootState } from 'data/rootReducer'
+
+const IconButtonCss = css`
+  border: 1px solid ${props => props.theme['grey100']};
+  border-radius: 8px;
+  margin-right: 12px;
+`
+const DownloadButton = styled(CSVLink)`
+  text-decoration: none;
+`
+const StyledIconButton = styled(IconButton)`
+  ${IconButtonCss};
+  color: ${props => props.theme['blue600']};
+`
+const SuccessIconButton = styled(IconButton)`
+  ${IconButtonCss};
+  color: ${props => props.theme.white};
+  background-color: ${props => props.theme.success};
```

Comment:
can we just use the color codes directly? I think this should be `blue600`

Reply:
The success color is unique (#00875A), should I use green600 (#00994C)